# metabolomics				
Tools	Implementation	Ease of use	Software availability	
GC-MS 				
CorrectOverloadedPeaks	R	difficult	https://cran.r-project.org/package=CorrectOverloadedPeaks	
eRah	R	difficult	http://cran.r-project.org/package=erah	
ADAP-GC 3.0	R	difficult	https://drive.google.com/drive/folders/0B8m21OI0wOSmeWFzZ3JxUVg0bms
GC–MS SPA	Manual	medium	NA	
flagme	R	difficult	https://bioconductor.org/packages/release/bioc/html/flagme.html	
AnalyzeIMS	MATLAB	difficult	https://github.com/BioMEMS/AnalyzeIMS	
MetabQ	R	difficult	http://metabolomics.auckland.ac.nz/index.php/projects/26	
PYQUAN	Python	difficult	https://github.com/lycopoda/Pyquan	
2D GC-MS				
GC2MS	Webserver	easy	http://gc2ms.web.cmdm.tw/index.php/research	
msPeakG	R	difficult	http://mrr.sourceforge.net/download.html	
osd	R	difficult	http://metabolomicsplatform.com/applications/	
SpectralAnalysis	Matlab	difficult	https://github.com/AlanRace/SpectralAnalysis	
NMR				
NMRPro	Web-based	easy	http://mamitsukalab.org/tools/nmrpro/	
SpiNCouple	Web-based	easy	http://emar.riken.jp/spincpl	
ChemSMP 	Python	difficult	http://nrc.iisc.ernet.in/hsa/gft.htm	
PROMEB	Python	difficult	http://nrc.iisc.ernet.in/hsa/gft.htm	
SpeckTackle	JavaScript	easy	https://bitbucket.org/sbeisken/specktackle	
NMRmix	Python	difficult	http://www.nmrfam.wisc.edu/software	
jsNMR	JavaScript	easy	http://nmr.au.dk/software/jsnmr/	
Targeted				
RIPPER	Java	medium	http://license.umn.edu/technologies/20120293_mass-spectrometry-software-improves-data-evaluation
MRMAnalyzer	R	difficult	http://www.metabolomics-shanghai.org/softwaredetail.php?id=34	
MetDIA	R	difficult	http://www.metabolomics-shanghai.org/softwaredetail.php?id=40	
Untargeted				
i-MetQ	.Net 	easy	http://ms.iis.sinica.edu.tw/comics/Software_iMet-Q	
NINA	Windows	medium	http://www.sciencedirect.com/science/article/pii/S0731708516304381	
MSCombine	R	difficult	https://cran.r-project.org/web/packages/MScombine/index.html	
Isotopes				
IsotopicLabelling 	R	difficult	https://github.com/RuggeroFerrazza/IsotopicLabelling	
MID Max	Python	difficult	https://github.com/dmccloskey/MDV_utilities	
geoRge	R	difficult	https://github.com/jcapelladesto/geoRge	
MIA	C++	difficult	http://massisotopolomeanalyzer.lu/	
ICT	Perl	difficult	https://github.com/jungreuc/isotope_correction_toolbox/	
QC				
LacaScore	NA	na	http://link.springer.com/article/10.1007/s11306-016-1038-1	
QCScreen	Windows, Python, R	difficult	https://sourceforge.net/projects/qcscreen/	
LC-MS pre-processing				
MET-XAlign	C++, Microsoft.NET Framework	medium	http://bioinfo.noble.org/manuscript-support/met-xalign/	
CCSWA	R	difficult	NA	
MetNormalizer	R	easy	http://www.metabolomics-shanghai.org/	
MetTailor	R	difficult	https://sourceforge.net/projects/mettailor/	
Gross and systematic error correction	NA	na	http://pubs.acs.org/doi/abs/10.1021/acs.analchem.5b03912	
GridMass	Java	easy	http://bioinformatica.mty.itesm.mx/GridMass	
BatchCorrMetabolomics	R	difficult	https://github.com/rwehrens/BatchCorrMetabolomics	
Annotation				
1-SToP	C#, R 	difficult	http://pubs.acs.org/doi/abs/10.1021/acs.analchem.6b02479	
iMet	Webserver	easy 	http://imet.seeslab.net./ 	
InterpretMSSpectrum	R	difficult	https://cran.r-project.org/package=InterpretMSSpectrum	
Elder	R	difficult	http://https//sourceforge.net/projects/metaopen/files/EIder/	
MSFinder	C#	easy	http://prime.psc.riken.jp/Metabolomics_Software/MS-FINDER/index.html 
MetFamily	R, Webserver	easy	http://msbi.ipb-halle.de/MetFamily/	
Greazy	.Net, R	difficult	http://tiny.cc/bumbershoot-vc12-bin64 	
tMICE	NA	na	NA	
Library/ Database				
WEIZMASS 	R	difficult	NA	
Secondary Metabolite Bioinformatics Portal	Web-based	easy	http://www.secondarymetabolites.org/	
MINEs	Webserver	easy	http://minedatabase.mcs.anl.gov/#/home	
MetCCS	Webserver	easy	http://www.metabolomics-shanghai.org/softwaredetail.php?id=108	
GlycoMob	Web-based	easy	http://www.glycomob.org/	
XMetDB	Webserver	easy	http://www.xmetdb.org/xmetdb	
Phenol-Explorer	Web-based	easy	NA	
Statistical				
SMART	R	difficult	http://www.stat.sinica.edu.tw/hsinchou/metabolomics/SMART.htm	
RANSAC-PLS	R	difficult	http://www.sciencedirect.com/science/article/pii/S1389172316000098	
tmod	R	difficult	https://cran.r-project.org/web/packages/tmod/index.html 	
metaCCA	MATLAB	difficult	https://github.com/aalto-ics-kepaco 	
metICA	R	difficult	https://github.com/daniellyz/MetICA 	
Cardinal	R	difficult	http://cardinalmsi.org/	
Pathways				
Subpathway-CorSP	R	difficult	NA	
KeyPathwayMiner	Web-based	easy	http://tomcat.compbio.sdu.dk/keypathwayminer/	
PathVisioRPC	Java	medium	http://projects.bigcat.unimaas.nl/pathvisiorpc/	
WikiPathways	R	difficult	https://github.com/egonw/rwikipathways	
PaxtoolsR	R	difficult	http://www.bioconductor.org/packages/paxtoolsr	
Escher	Web	easy	https://escher.github.io/	
Network				
MetaMapR	R, Web-based	easy	http://dgrapov.github.io/MetaMapR/	
GAM	Web-based	easy	https://artyomovlab.wustl.edu/shiny/gam/	
DiffCorr	R	difficult	https://cran.r-project.org/web/packages/DiffCorr/index.html	
Visualization				
ili	R	difficult	https://github.com/ili-toolbox/ili	
msIQUANT	Windows	medium	http://ms-imaging.org/wp/paquan/	
BatMass	Windows	easy	http://www.batmass.org/	
Heatmapper	Webserver	easy	http://www.heatmapper.ca/	
MSIdV	Python	difficult	https://sourceforge.net/projects/msidv/	
Global				
MAIT	R	difficult	https://bioconductor.org/packages/release/bioc/html/MAIT.html	
MetaboAnalyst 3.0	Web, R	easy	http://www.metaboanalyst.ca/	
Galaxy-M	MATLAB, Galaxy	difficult	https://github.com/Viant-Metabolomics/Galaxy-M	
PIUMet	Webserver	easy	http://fraenkel-nsf.csbi.mit.edu/PIUMet/	
specmine	R	difficult	https://cran.r-project.org/web/packages/specmine/index.html	
OpenMS	Windows	easy	http://www.openms.de/	
Others				
ChemDataExtractor	Webserver	easy	http://chemdataextractor.org/	
ConceptMetab	Webserver	medium	http://conceptmetab.med.umich.edu/	
ReconMap	Webserver	easy	https://github.com/opencobra/cobratoolbox	
MBROLE 2.0	Webserver	easy	http://csbg.cnb.csic.es/mbrole2	
biosigner	R	difficult	https://bioconductor.org/packages/release/bioc/html/biosigner.html	
USR-Vis	Webserver	easy	http://usr.marseille.inserm.fr/	
CypRules	Webserver	easy	http://cyprules.cmdm.tw/	
LOBSTAHS	R	difficult	https://github.com/vanmooylipidomics/LOBSTAHS	
PredRet	Webserver	easy	http://predret.org/	
				
Name		Platform Dependency	Web	References
Data handling and preprocessing tools			
MetMSLine		LC-MS; R	http://wmbedmands.github.io/MetMSLine/ 	
MSPrep		MS, R	http://sourceforge.net/projects/msprep/	
MRM-DIFF		LC-MS; Windows	http://prime.psc.riken.jp/	
MRMPROBS		LC-MS; Windows	http://prime.psc.riken.jp/	
FragPred		MS; R	http://pattilab.wustl.edu/software/FragPred/index.php 	
IsoMS		LC-MS; R, Windows	www.mycompoundid.org/IsoMS	
MyCompoundID.org		MS; Web	http://mycompoundid.org/	
isoMETLIN		MS/MS; Web	http://isometlin.scripps.edu/	
IPO		LC-MS; R	https://github.com/glibiseller/IPO	
Massifquant/ Optimize-it		MS; Bioconductor	https://github.com/topherconley/optimize-it 	
IsoMS Quant		LC-MS; R, Windows	www.mycompoundid.org/IsoMS	
MUSCLE		LC-MS; C++	http://www.muscleproject.org/	
intCor		LC-MS; R	http://b2slab.upc.edu/software-and-downloads/intensity-drift-correction/
Peak-group-alignment		LC-MS, R	https://github.com/joewandy/peak-grouping-alignment	
Parametric time warping		LC-DAD, LC-MS; R	https://github.com/rwehrens/ptw 	
PeakANOVA		LC-MS; R	http://research.ics.aalto.fi/mi/software/peakANOVA/ 	
				
Statistical tools				
DeviumWeb		Any; R, Web	https://github.com/dgrapov/DeviumWeb 	
EigenMS		MS; R, MATLAB	http://sourceforge.net/projects/eigenms/	
RepExplore		Any; Web	http://www.repexplore.tk	
Normalyzer		Any; Web	http://quantitativeproteomics.org/normalyzer	
BioStatFlow		Any; Web	http://biostatflow.org/	
Annotation tools				
MS-DIAL		LC-MS; Windows	http://prime.psc.riken.jp/	
MET-COFEA		LC-MS, Windows	http://bioinfo.noble.org/ manuscript-support/met-cofea/	
MetAssign		LC-MS; Java	http://mzmatch.sourceforge.net/MetAssign.php 	
MS2Analyzer		MS/MS; Java	http://fiehnlab.ucdavis.edu/projects/MS2Analyzer/	
Metabolome searcher		-, Web	http://procyc.westcent.usu.edu/cgi-bin/MetaboSearcher.cgi	
MAGMa		MS/MS; Java	https://www.emetabolomics.org/ 	
mzGroupAnalyzer		LC-MS; MATLAB	http://www.univie.ac.at/mosys/software.html

Cosmiq		LC-MS, GC-MS; R	http://www.bioconductor.org/packages/release/bioc/html/ cosmiq.html	
Feature Credential		MS/MS; R	http://pattilab.wustl.edu/software/credential/	
ProbMetab		LC-MS; R	http://labpib.fmrp.usp.br/methods/probmetab/	
HR3		GC-MS, LC-MS; Windows	www.metalign.nl	
MAIT 		MS, R	http://b2slab.upc.edu/software-and-downloads/metabolite-automatic-identification-toolkit/ 
BioSM		MS/MS; Java	http://metabolomics.pharm.uconn.edu	
BioSMXpress		-	http://engr.uconn.edu/~rajasek/BioSMXpress.zip	
CFM-ID		ESI-MS/MS; Web	http://cfmid.wishartlab.com/	
MIDAS		MS/MS; Web	http://midas.omicsbio.org	
GS-align		-; C++	http://www.glycanstructure.org/gsalign	
POCHEMON		LC-MS; MATLAB	http://www.ru.nl/science/analyticalchemistry/research/software/	
RAMClustR		MS/MS; R	https://github.com/cbroeckl/RAMClustR	
LipidPro		MS/MS; Windows	http://www.neurogenetics. biozentrum.uni-wuerzburg.de/en/project/services/l ipidpro/
mzCloud		MS/MS; Web	https://www.mzcloud.org/ 	
Pathway and networks analysis and biological interpretation tools		
PathCaseMAW		Any; Web	http://nashua.case.edu/PathwaysMAW/Web	
MINE		-; Java, Perl, Python	http://minedatabase.mcs.anl.gov/#/home 	
PathWhiz		-	http://smpdb.ca/pathwhiz	
TrackSM		MS/MS; MATLAB	http://metabolomics.pharm.uconn.edu/?q=Software.html	
MarVis-Suite		Any, Java	http://marvis.gobics.de	
InCroMAP		Any; Java	http://www.cogsys.cs.uni-tuebingen.de/software/InCroMAP	
iPEAP		Any; Windows; Java’; R	http://www.tongji.edu.cn/~qiliu/ipeap.html 	
kpath		Any; Web	http://browser.kpath.khaos.uma.es/	
Pathomx		Any; Python; MATLAB; R	http://pathomx.org/	
MetaMapR		Any; R, Web	http://dgrapov.github.io/MetaMapR/	
Metabnet		LC-MS; R	https://sourceforge.net/projects/metabnet/ 	
MetaNET		Any; Galaxy	http://metanet.osdd.net	
Funrich		-, Web	http://www.funrich.org/ 	
Network portal		Any; Web	http://networks.systemsbiology.net	
SimIndex		Any; Python	http://tyolab.northwestern.edu/tools/ 	
BiNChE		-	http://www.ebi.ac.uk/chebi/tools/binche/	
PlantSEED		-	http://bioseed.mcs.anl.gov/~seaver/FIG/seedviewer.cgi?page=PlantSEED	
SPICA		LC-MS	http://cmcr.columbia.edu/metabolomics /informaticstools.html	
KiMoSys		Any; Web	http://kimosys.or	
Integrated Interactome System	Any; Web	http://www.lge.ibi.unicamp.br/lnbio/IIS/	
MetaDB		GC-MS, LC-MS, R	https://github.com/rmylonas/MetaDB 	
NMR-based Tools				
Focus		NMR; MATLAB	http://www.urr.cat/FOCUS	
Metabnorm		NMR; R	http://sourceforge.net/projects/metabnorm/ 	
BATMAN		NMR; R	http://batman.r-forge.r-project.org/	
Bayesil		NMR Web	http://bayesil.ca/	
SENECA		NMR; Java	http://sourceforge.net/projects/seneca/	
COLMAR		NMR; Web	http://spin.ccic.ohio-state.edu/index.php/colmar	
1H(13C)-TOCCATA		NMR; Web	http://spin.ccic.ohio-state.edu/index.php/toccata2/index	
mQTL.NMR		NMR; R	http://www.ican-institute.org/tools/	
MVAPACK		NMR; MATLAB	http://bionmr.unl.edu/mvapack.php	
ChemoSpec		NMR; R	http://cran.r-project.org/web/packages/ChemoSpec/ 	
GC-MS-based tools				
MetaMS		GC-MS; R	http://www.bioconductor.org/packages/release/bioc/html/metaMS.html 
Maui-VIA		GC-MS; Java	http://bimsbstatic.mdc-berlin.de/kempa/software/kempaSoftware.html	
PScore		GC-MS; R	http://raphaelaggio.github.io/	
BiPACE 2D		2D GC-MS; 	http://maltcms.sf.net/ 	
Library, databases and others			
HAMMER		MS/MS; Java, Python	http://www.biosciences-labs.bham.ac.uk/viant/hammer/)	
TAPIR		MS/MS; Python	https://github.com/msproteomicstools/msproteomicstools	
TMDB		-	http://pcsb.ahau.edu.cn:8080/TCDB/	
BioPhytMol database		-	http://ab-openlab.csir.res.in/biophytmol/	
EssOilDB		-	http://nipgr.res.in/Essoildb/	
mVOCs		-	http://bioinformatics.charite.de/mvoc	
domdb		-, SQL	https://github.com/joefutrelle/domdb 	
PhenoMeter		-, Web	https://www.metabolome-express.org/phenometer.php 	
T3DB		-, Web	www.t3db.ca 	
SwissLipids		MS, Web	http://www.swisslipids.org/	
Metabolonote			http://metabolonote.kazusa.or.jp/	
KOMICS		-	http://www.kazusa.or.jp/komics/	
QTREDS		-; Ruby, Rails	http://qtreds.crs4.it 	
MASTR-MS		-	https://mastr-ms.readthedocs.org/en/latest/	
Yabi			http://ccg.murdoch.edu.au/yabi/	
jmzTab		-	http://mztab.googlecode.com	
PolySearch2		-; Web	http://polysearch.ca 	
SpeckTackle		MS/MS, NMR; Java	https://bitbucket.org/sbeisken/specktackle 	
Metabolite Imager		-, Web	www.metaboliteimager.com	
EXIMS		MALDI-MS, -	http://exims.sourceforge.net/	
BioMet Toolbox 2.0		-, Web	http://biomet-toolbox.org/	
Multifunctional tools				
XCMS Online		Web	https://xcmsonline.scripps.edu/	
Mass++		Any, Java	http://www.first-ms3d.jp/english/ 	
MASSyPup		LC-MS, GC-MS; Linux	http://www.bioprocess.org/massypup	
MetaboNexus		Any; Windows, Java	http://www.sph.nus.edu.sg/index.php/ research-services/research-centres/ceohr/metabonexus
MetaboLyzer		LC-MS; Python	https://sites.google.com/a/georgetown.edu/fornace-lab-informatics/home/ metabolyzer
Workflow4Metabolomics		LC-MS, GC-MS, NMR; Galaxy	http://workflow4metabolomics.org	
Haystack		LC-MS; Web	http://binf-app.host.ualr.edu/haystack/	
ALLocator		LC-MS,13C; Web	https://allocator.cebitec.uni-bielefeld.de

MeKO		GC-MS; Web	http://prime.psc.riken.jp/meko/	
MassCascade		LC-MS, KNIME	https://bitbucket.org/sbeisken/masscascadeknime/wiki/Home	
				
				
				
